lib/mix/lib/mix/tasks/profile.eprof.ex

```diff
@@ -95,29 +95,53 @@ defmodule Mix.Tasks.Profile.Eprof do
   `Mix.Tasks.Profile.Cprof` that uses `:cprof` and has a low performance degradation effect.
   """
 
-  @switches [parallel: :boolean, require: :keep, eval: :keep, config: :keep, matching: :string,
-             halt: :boolean, compile: :boolean, deps_check: :boolean, calls: :integer,
-             time: :integer, sort: :string, start: :boolean, archives_check: :boolean,
-             warmup: :boolean, elixir_version_check: :boolean, parallel_require: :keep]
+  @switches [
+    parallel: :boolean,
+    require: :keep,
+    eval: :keep,
+    config: :keep,
+    matching: :string,
+    halt: :boolean,
+    compile: :boolean,
+    deps_check: :boolean,
+    calls: :integer,
+    time: :integer,
+    sort: :string,
+    start: :boolean,
+    archives_check: :boolean,
+    warmup: :boolean,
+    elixir_version_check: :boolean,
+    parallel_require: :keep
+  ]
 
   def run(args) do
     {opts, head} =
-      OptionParser.parse_head!(args,
+      OptionParser.parse_head!(
+        args,
         aliases: [r: :require, p: :parallel, e: :eval, c: :config],
```

Comment:
Can you please move the aliases to a module attribute as well (similar to `@switches`) so everything fits a single line? Thanks!

Reply:
So something like this?
```elixir
  @aliases [
    r: :require,·
    p: :parallel,·
    e: :eval,·
    c: :config
    ]
  
   def run(args) do
      {opts, head} =  OptionParser.parse_head!(args, aliases: @aliases, strict: @switches)
   
        Mix.Tasks.Run.run(
          ["--no-mix-exs" | args],
          opts,
          head,
          &profile_code(&1, opts),
          &profile_code(File.read!(&1), opts)
        )
   end
```